image/rhel/konflux.Dockerfile

```diff
@@ -81,9 +127,20 @@ COPY --from=rpm-installer "$FINAL_STAGE_PATH" /
 
 COPY --from=ui-builder /go/src/github.com/stackrox/rox/app/ui/build /ui/
 
-# TODO: move this to go-builder stage. Will be done as part of the same ROX-20160.
-COPY --from=rocksdb-builder /staging/librocksdb.a /lib/rocksdb/librocksdb.a
-COPY --from=rocksdb-builder /staging/include /lib/rocksdb/include
+COPY --from=go-builder /go/src/github.com/stackrox/rox/app/image/rhel/bin/migrator /stackrox/bin/
+COPY --from=go-builder /go/src/github.com/stackrox/rox/app/image/rhel/bin/central /stackrox/
+COPY --from=go-builder /go/src/github.com/stackrox/rox/app/image/rhel/bin/compliance /stackrox/bin/
+COPY --from=go-builder /go/src/github.com/stackrox/rox/app/image/rhel/bin/roxctl* /assets/downloads/cli/
+COPY --from=go-builder /go/src/github.com/stackrox/rox/app/image/rhel/bin/kubernetes-sensor /stackrox/bin/
+COPY --from=go-builder /go/src/github.com/stackrox/rox/app/image/rhel/bin/sensor-upgrader /stackrox/bin/
+COPY --from=go-builder /go/src/github.com/stackrox/rox/app/image/rhel/bin/admission-control /stackrox/bin/
+COPY --from=go-builder /go/src/github.com/stackrox/rox/app/image/rhel/static-bin/* /stackrox/
+RUN GOARCH=$(uname -m) ; \
+    case $GOARCH in x86_64) GOARCH=amd64 ;; aarch64) GOARCH=arm64 ;; esac ; \
+    ln -s /assets/downloads/cli/roxctl-linux-$GOARCH /stackrox/roxctl ; \
+    ln -s /assets/downloads/cli/roxctl-linux-$GOARCH /assets/downloads/cli/roxctl-linux
+
+# TODO: rhacs-main/Dockerfile.in?ref_type=heads#L107-122
```

Comment:
As discussed this is to be addressed in a separate PR.
While looking at https://github.com/stackrox/stackrox/blob/master/image/rhel/fetch-stackrox-data.sh & https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-main/pre-build-script?ref_type=heads, I was wondering why we do not include the Istio CVE list in upstream images.

Reply:
Looks like it was removed upstream about a year ago. See https://github.com/stackrox/stackrox/pull/4341

Probably downstream was forgotten.

Asked in Slack https://redhat-internal.slack.com/archives/C033SKHG9CZ/p1712598818791699